docs/testbed/README.testbed.Testbed_v2.hld.md

```diff
@@ -0,0 +1,127 @@
+# Testbed V2 Design
+# High Level Design Document
+
+
+# Table of Contents
+  * [About this Manual](#about-this-manual)
+  * [Requirement Overview](#Requirement-Overview)
+  * [Components](#Components)
+  * [Database Schema](#Database-Schema)
+  * [Implementation Plan](#Implementation-Plan)
+
+
+## About this Manual
+This documentation provides general information about the Testbed V2 feature implementation for `sonic-mgmt`. Testbed V2 aims to support multi-DUTs deployment with dynamic VLAN assignment and inter-DUTs link state propagation.
+
+## Requirement Overview
+* Maintain compatibility with existing testbed.
+* Support testbed/topology with multiple DUTs that have inter-DUTs connections.
+* Support flexible VLAN assignment.
+
+## Components
+* `test user`
+* `connection_db`
+  * Redis databases running on certain test server hosting connection-related metadata.
+* `servercfgd`
+  * running on the same server as `connection_db`
+  * a RPC server that is responsible for `connection_db` initial setup and provision.
+* `labcfgd`
+  * running on root/fanout switches
+  * subscribe to keyspace events of `connection_db` and act accordingly.
+
+## Database Schema
+* `LAB_CONNECTION_GRAPH_VERSIONS`
+```
+; ZSET that stores md5sum values of connection graph files that are used to provision `connection_db`
+key                              = LAB_CONNECTION_GRAPH_VERSIONS
+```
+* `LAB_META`
```

Comment:
I think if we plan on adding more info about lab metadata this name is OK, but I feel like right now this is more DB_META?

Reply:
Agreed, will modify then.